crates/evm/evm/src/executors/invariant/replay.rs

```diff
@@ -34,7 +34,7 @@ pub fn replay_run(
 ) -> Result<Vec<BaseCounterExample>> {
     // We want traces for a failed case.
     if executor.inspector().tracer.is_none() {
-        executor.set_tracing(true, false);
+        executor.set_tracing(Some(TraceMode::Call));
```

Comment:
we could add `TraceMode::None` which I think is more ergonomic than doing Option,

but maybe not so we can have an easy conversation into `TracingInspectorConfig`